### PR TITLE
Adding server commands for watching and unwatching

### PR DESCRIPTION
**Pull request by charj**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,54 +1,60 @@
-import { WatchOptions } from 'chokidar';
+import { WatchOptions } from "chokidar"
```
Comment (charj) [inline src/config.ts:1]: These are just formatting improvements done by the linter..

File: src/utils.ts [hunk 2]
```diff
@@ -8,7 +8,6 @@ export function fileExistsInPath(folder: string, file: string): boolean {
 }
 
 export function checkIfValidResource(resource_path: string): boolean {
-  const fx_manifest = path.join(resource_path, 'fxmanifest.lua');
```
Comment (charj) [inline src/utils.ts:11]: Another linting issue

**Review by mrdiamonddirt — APPROVED** [review]
looks good to me 👍

**Review by mrdiamonddirt — APPROVED** [review]
👍

**Pull request merged**